controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -146,25 +142,49 @@ def _test_copy_to_existing_volume_from_snapshot_error(self, xcli_exception, expe
         with self.assertRaises(expected_array_exception):
             self.mediator.copy_to_existing_volume_from_source("vol", "snap", 0, 0)
 
+    def test_copy_to_existing_volume_from_volume_failed_volume_not_found(self):
```

Comment:
how is it different than `test_copy_to_existing_volume_from_snapshot_failed_snapshot_not_found`?

Reply:
removed. it added no coverage